packages/gestalt/src/IconButton.tsx

```diff
@@ -30,6 +32,11 @@ type Props = {
    * Indicates whether this component displays a menu, such as Dropdown, or a dialog, like Popover, Modal or ModalAlert. See the [Accessibility](https://gestalt.pinterest.systems/web/iconbutton#ARIA-attributes) guidelines for details on proper usage.
    */
   accessibilityPopupRole?: 'menu' | 'dialog';
+  /**
+   * Indicates whether this component is hosted in a light or dark container.
+   * Used for improving focus ring color contrast.
+   */
+  focusColor?: 'lightBackground' | 'darkBackground';
```

Comment:
Can we document this in the docs?

Reply:
Are we supporting this in IconButtonLink?

Reply:
Added support and examples!